ui/src/pages/insightDetail/components/eventAggregator/styles.module.less

```diff
@@ -349,13 +346,12 @@
         }
       }
 
```

Comment:
Removing `padding: 16px;` could cause content to be too close to the edges. Consider adding it back if spacing is needed.